src/client/server.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const path = require("path");
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+// set up rate limiter: maximum of five requests per minute
+var RateLimit = require("express-rate-limit");
+var limiter = RateLimit({
+  windowMs: 1 * 60 * 1000, // 1 minute
+  max: 5,
+});
+
+// apply rate limiter to all requests
+app.use(limiter);
```

Comment:
Q: Das Rate Limit gilt für sämtliche Requests in der gesamten Applikation. 5 Klicks pro Minute und IP sind warscheinlich (zu) knapp bemessen, oder? Login-Seite -> Login -> Daten laden -> Daten laden -> Daten laden ... etc

Reply:
Klar, ich schraube den Wert hoch.